### PR TITLE
Schema of String and Wrapper Objects, OpenApi version to 3.0.0

### PR DESCRIPTION
**Pull request by ralphdoe**
Changes in the Schema definitions of Wrapper and String objects
OpenApi default version to 3.0.0
UserResource Test Added

### CODE REVIEW

File: modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java [hunk 1]
```diff
@@ -375,4 +376,189 @@ public void testCompleteOperation() {
...
         assertEquals(extractedYAML, expectedYAML);
     }
+
+    @Test(description = "reads the pet resource from sample")
+    public void testCompleteUserResource() {
+        String openApiYAML = readIntoYaml(UserResource.class);
+        int start = 0;
+        int end = openApiYAML.length() - 1;
+        String extractedYAML = openApiYAML.substring(start, end);
+        String expectedYAML = "openapi: 3.0.0-rc2\n" +
```
Comment (frantuma) [inline modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java]: change to 3.0.0

File: modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java [hunk 2]
```diff
@@ -375,4 +376,189 @@ public void testCompleteOperation() {
                 "          - \"available,pending,sold\"";
         assertEquals(extractedYAML, expectedYAML);
     }
+
+    @Test(description = "reads the pet resource from sample")
```
Comment (frantuma) [inline modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java]: rename description

Comment (ralphdoe) [inline modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java]: Done!

File: modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java (hunk 1 above)
Comment (ralphdoe) [inline modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java]: Done and also changed some other version in a json file.

**Pull request merged**